packages/app-page-builder/src/types.ts

```diff
@@ -692,7 +692,7 @@ export type PbRenderElementPluginArgs = {
 // TODO: at some point, convert this into an interface, and use module augmentation to add new properties.
 export type EventActionHandlerCallableState<TState = PbState> = PbState<TState> & {
     getElementById(id: string): Promise<PbEditorElement>;
-    getElementTree(element?: PbEditorElement): Promise<any>;
+    getElementTree(props?: { element?: PbEditorElement }): Promise<any>;
```

Comment:
@neatbyte-vnobis is there a reason to convert this to a `props` object? I must be missing something, but it doesn't look like you're using anything other than the `element`, so not sure you need this 🤔 

And secondary, why `Promise<any>`? If you have a different form of `element` coming back from `getElementTree`, maybe it's better to add support for generic type on that function.

Reply:
@Pavel910 I have updated the types

Not only `element` is used (in recursion also `path` is passed).
Due to composeAsync inner working only one parameter can be passed, so `getElementTree(element, path)` was replaced with `getElementTree({ element, path })`.

I have created a new type inside types (as there were multiple definitions with different interfaces in types and EventActionHandlerProvider.tsx)

And yes, no need for `any` there